README.md

```diff
@@ -27,115 +31,164 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
 will start a batch of tests whenever a new pull request is open in the "builds"
 and "versions" repositories by an authorized user, when those pull requests are
 updated with new commits or when an authorized user comments the phrase
-"start tests". Builds and Pylint validation can be triggered independently with
-the phrases "start build" and "start pylint", respectively.
-
-### Release process
-
-A Jenkins job will trigger periodically to update the packages' versions, create
-release notes and execute a build with all packages available, by executing the
-corresponding commands from the "builds" repository scripts. Commits will be
-created for the "versions" and "open-power-host-os.github.io" repositories and
-pushed to the designated GitHub user's repository. Ideally, this should be a
-"bot" user. Pull requests for those commits can be created manually and merged
-into the main organization/user repository, if desired.
-
-Note: The default is to run those periodic builds once a week (every Wednesday
-at 11 AM).
+"start tests". Currently, tests are: building of all packages, Pylint validation
+of Python code, yamllint validation of all package metadata YAML files and rpmlint
+validation of all RPM packages specification files. These tests can be triggered
+independently with the phrases "start build", "start pylint", "start yamllint"
+and "start rpmlint" respectively.
+
+### Periodic builds process
+
+A Jenkins job will trigger periodically to check the versions of the packages
+available in OpenPOWER Host OS GitHub organization, update the corresponding
+packages' versions in the "versions" git repository, create a build with all
+packages available and create release notes, by executing the corresponding
+commands from the build scripts in the "builds" git repository. Commits will
+be created for the local "versions" and "open-power-host-os.github.io" git
+repositories and pushed to the designated GitHub user's git repository. Ideally,
+this should be a "bot" user. Pull requests for those commits can be created
+manually and merged into the main organization/user git repository, if desired.
+
+Note: The default is to execute those periodic builds once a week (every Wednesday
+at 11 AM UTC-03:00).
```

Comment:
OH> Having a setup that works for other teams is not an optional plus, it is a goal that we must work towards.

Why? Why other teams need to submit pull requests to a git hub organization and have builds triggered automatically? Why other teams need to create release notes? Why other teams need to update versions? The only feature I really see other teams need is to build OP Host OS and they don't even need Jenkins to do this, it can be done much more easily in the command line! Do we know of at least ONE team which uses this Jenkins infrastructure or wants to?

OH> This can't be done all at once, so the first step was to make this work for us. The future steps will be to make this easy for everybody else.

If the first step is to make it work for us (and this makes sense), why are we spending time in a doc which is not for us? Why not build a doc which is for us *first* and later work on a generic doc which would work for others? This is not consistent.

OH> The specific configuration for our use case should be in an internal document (it does not exist yet) 

So we are sacrificing having a full documentation about how to replicate *our* infrastructure in case of disaster to support a documentation which is not used by anyone currently?